### DISCUSSION TITLE
proxy

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#4430] [Help]
Hi,

I have just done an update and stumbled across a problem.  It seems like instead of using envoy budi now uses nginx as a proxy, but for some reason it is not working:

I believe it could be due to this error in the proxy container:
2022/02/11 12:29:08 [emerg] 1#1: host not found in upstream "app-service.budibase.svc.cluster.local" in /etc/nginx/nginx.conf:47
nginx: [emerg] host not found in upstream "app-service.budibase.svc.cluster.local" in /etc/nginx/nginx.conf:47

so the container restarts every minute? what do you think?

M

**Comment by marco-santulli** [comment]
sorry just saw there is a bug opened for this just ignore thx

**Comment by shogunpurple** [comment]
Hi @marco-santulli - could you try pulling the latest image now? It should have been fixed - there was an issue where the docker image got pushed to the the wrong tag.

**Comment by marco-santulli** [comment]
Hi,

small feedback:
1. budi hosting --update doesn't pull the latest one yet, but used docker pull
2. it is working if I go to the builder, but if I go to the "main" site it doesnt at least for my config:
https://host/ redirects to https://host:10000/ which is not ok for me because I have my own nginx proxy, (note I have MAIN_PORT=8000
in the .env file) but if I go to:
https://host/builder, then it works.

I will keep investigating and report

**Reply by shogunpurple** [reply]
Thanks @marco-santulli - we will have a look at this ASAP. Seeing the same issue

**Reply by shogunpurple** [reply]
Found it - pushing fix now and you should be able to pull the latest image very soon

**Reply by marco-santulli** [reply]
I don't see it am I doing something wrong or it's not available yet?

**Reply by shogunpurple** [reply]
Hi @marco-santulli - that should be updated now - we just rolled out the release. Thank you!

**Reply by marco-santulli** [reply]
I have pulled the  new version but I still see the same behaviour.  I get this:

https://myserver.mydomain.eu/   301 -->
https://myserver.mydomain.eu/builder   301 -->
http://myserver.mydomain.eu:10000/builder/   307 -->  (note no https)
https://myserver.mydomain.eu:10000/builder/   FAILED Net:err -->  (note no https)

I think the issue is in that second step, where it should go to https://myserver.mydomain.eu/builder/ with a slash at the end.  That works.  And for some reason when it gets to the /builder it wants to go to port 10000 (I have the following in .env):

# Use the main port in the builder for your self hosting URL, e.g. localhost:10000
MAIN_PORT=8000
